src/Yoti.Auth/Activity.cs

```diff
@@ -13,7 +14,14 @@ namespace Yoti.Auth
 {
     internal class Activity
     {
-        private static YotiUserProfile _yotiUserProfile;
+        private YotiUserProfile _yotiUserProfile;
+        private YotiProfile _yotiProfile;
+
+        public Activity(YotiProfile yotiProfile, YotiUserProfile yotiUserProfile)
```

Comment:
I don't understand this. Is this for backward compatibility?
If so, given that the class is internal, you should really have a constructor that accepts a `YotiProfile` and convert any `YotiUserProfile` into `YotiProfile` before instantiating `Activity` instead.